recipes/rtoml/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "rtoml" %}
+{% set version = "0.8.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/rtoml-{{ version }}.tar.gz
+  sha256: 6d55fa9df88256c6762f01f18b8bd3f6dc1836600ea8655eb3719a2b8f3df764
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip >=21.0.1
+    - python >=3.7
+    - setuptools_rust
```

Comment:
```suggestion
    - setuptools-rust
```

Reply:
@goanpeca Thank you very much for helping with this.